src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,106 @@
+package wolox.training.models;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    private Long userId;
+
+    @Column(nullable = false, unique = true)
+    private String userUsername;
+
+    @Column(nullable = false)
+    private String userName;
+
+    @Column(nullable = true)
+    private LocalDate userBirthday;
```

Comment:
You can remove the prefix "user" from the attributes so that it is not redundant when invoking the fields: `user.getUserUsername`
```suggestion
    private Long id;

    @Column(nullable = false, unique = true)
    private String userUsername;

    @Column(nullable = false)
    private String userName;

    @Column(nullable = true)
    private LocalDate userBirthday;
```